go.mod

```diff
@@ -222,7 +222,7 @@ require (
 	github.com/go-openapi/swag v0.22.3 // indirect
 	github.com/gocql/gocql v1.2.1
 	github.com/golang-jwt/jwt v3.2.2+incompatible // indirect
-	github.com/golang-jwt/jwt/v4 v4.2.0
+	github.com/golang-jwt/jwt/v4 v4.4.2
```

Comment:
Would you mind moving this out of the "// indirect" dependency block to the one above?